src/policy/inputs.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+// NOTE: This file is intended to be customised by the end user, and includes only local node policy logic
+
+#include "policy/policy.h"
```

Comment:
@jonasschnelli You can include `script/standard.h` here instead of `policy/policy.h` to cut all ties :)

Mabye move the new include further down next to `script/interpreter.h`.